CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 2.0.0
+
+The is a major release
+
+* [TK-369](https://tickets.puppetlabs.com/browse/TK-369) Move Jetty dependency to 9.4.1
```

Comment:
ahhhhh. very well then. i think as of now that is java 7 support ending. unless more changes land before we actually cut 2.0.0. but i can mention that no APIs have changed.